src/components/InView/InViewLoader.tsx

```diff
@@ -10,7 +10,7 @@ export function InViewLoader({
   style,
 }: {
   children: React.ReactNode;
-  loadFn: () => void;
+  loadFn: () => any | Promise<any>;
   loadCondition: boolean;
```

Comment:
Just a suggestion, not necessarily needs to happen, but I think renaming these props to go with how we name them in other places would work best here
```suggestion
  onLoad: () => Promise<void>;
  run: boolean;
```